src/index.ts

```diff
@@ -43,7 +43,7 @@ export = function css({ filename = '[name].css', allChunks = false, sourceMap =
         rules: get(this, 'module.rules', []).concat([{
           test,
           use: extractCss ?
-            extractText.extract({ fallbackLoader: loaders[0], loader: loaders.slice(1) }) : 
+            extractText.extract({ fallback: loaders[0], use: loaders.slice(1) }) :
```

Comment:
Hmm, can `use` be an Array? The docs say it's "string / object", not sure what they mean by "object", perhaps it needs to be `use: { loaders: [...] }`? Could you find this out?